lib/runners/processor/metrics_fileinfo.rb

```diff
@@ -0,0 +1,64 @@
+module Runners
+  class Processor::MetricsFileInfo < Processor
+    Schema = _ = StrongJSON.new do
+      # @type self: SchemaClass
+
+      let :runner_config, Schema::BaseConfig.base
+      let :issue, object(
+        line_of_code: integer?,
+        last_commit_datetime: integer
+      )
+    end
+
+    def analyzer_version
+      @analyzer_version ||= extract_version!("wc")
+    end
+
+    # This analyser use git metadata (.git/).
+    def use_git_metadata_dir?
+      true
+    end
+
+    def analyze(changes)
+      Results::Success.new(guid: guid, analyzer: analyzer).tap do |result|
+        changes.changed_paths.each do |path|
+          result.add_issue(generate_issue(path))
+        end
+      end
+    end
+
+    private
+    def generate_issue(path)
+      filepath = relative_path(path).to_s
+      loc = analyze_line_of_code(filepath)
+      last_commit_iso, last_commit_epoch = analyze_last_commit_datetime(filepath)
+
+      Issue.new(
+        path: relative_path(path),
```

Comment:
[nitpick] We can avoid `relative_path` calls twice:

```suggestion
      filepath = relative_path(path)
      loc = analyze_line_of_code(filepath.to_path)
      last_commit_iso, last_commit_epoch = analyze_last_commit_datetime(filepath.to_path)

      Issue.new(
        path: filepath,
```